lib/actions/build.coffee

```diff
@@ -0,0 +1,224 @@
+Promise = require('bluebird')
+tar = require('tar-stream')
+dockerBuild = require('resin-docker-build')
+resolve = require('resin-bundle-resolve')
+klaw = require('klaw')
+path = require('path')
+fs = require('mz/fs')
+_ = require('lodash')
+
+# When we do TLS enabled docker builds, node sometimes complains
+# about the certificates not being fully authorized, this stops
+# that.
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
```

Comment:
Definitely shouldn't be committed, not sure why it was still there (had to be there for my testing). It definitely should be something that the user turns on though, fully agree.